portfolio/pom.xml

```diff
@@ -23,6 +23,11 @@
       <version>4.0.1</version>
       <scope>provided</scope>
     </dependency>
+    <dependency>
+        <groupId>com.google.code.gson</groupId>
```

Comment:
super nit: looks like these are indented twice as much as the tags above, I would keep these consistent if possible

Reply:
> super nit: looks like these are indented twice as much as the tags above, I would keep these consistent if possible

Will adjust it in